modules/javafx.graphics/src/main/java/javafx/stage/StageStyle.java

```diff
@@ -66,5 +72,40 @@ public enum StageStyle {
      * NOTE: To see the effect, the {@code Scene} covering the {@code Stage} should have {@code Color.TRANSPARENT}
      * @since JavaFX 8.0
      */
-    UNIFIED
+    UNIFIED,
+
+    /**
+     * Defines a {@code Stage} style in which the client area is extended into the header bar area, removing
+     * the separation between the two areas and allowing applications to place scene graph nodes in the header
+     * bar area of the stage.
+     * <p>
+     * An extended window has the default window buttons (minimize, maximize, close), but no system-provided
+     * draggable header bar. Applications need to provide their own header bar by placing the {@link HeaderBar}
```

Comment:
this must be emphasized.

suggestion:  "Applications must provide their own header bar by placing a single instance of the {@link HeaderBar} at the top of the scene graph"

and may be explain why

"in order to avoid the platform decorations overlapping the scene nodes" or something to that effect.

what do you think?

Reply:
Good idea, I've updated the documentation. I also included a paragraph about the title text.

Reply:
thank you!  can you add the application title row to the table in the JEP please?